core/src/streams.rs

```diff
@@ -359,16 +359,26 @@ impl<'gc> NetStream<'gc> {
                         let codec = VideoCodec::from_u8(codec_id as u8);
 
                         match (video_handle, codec, data) {
-                            (Some(video_handle), Some(codec), FlvVideoPacket::Data(data))
+                            (Some(video_handle), Some(codec), FlvVideoPacket::Data(mut data))
                             | (
                                 Some(video_handle),
                                 Some(codec),
                                 FlvVideoPacket::Vp6Data {
                                     hadjust: _,
                                     vadjust: _,
-                                    data,
+                                    mut data,
                                 },
                             ) => {
+                                if codec == VideoCodec::ScreenVideo {
```

Comment:
I think the same applies to V2 Screen Video. I plan to add the decoder for that back, and it's basically fused together with the current V1 one.

Reply:
But we can change this once we actually have that decoder... :)

Reply:
I'll add that condition in then